wix/alacritty.wxs

```diff
@@ -0,0 +1,84 @@
+
+        <!-- Create directories -->
+        <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="ProgramFiles64Folder">
+                <Directory Id="AlacrittyProgramFiles" Name="Alacritty"/>
+            </Directory>
+            <Directory Id="AppDataFolder">
+                <Directory Id="AlacrittyAppData" Name="alacritty"/>
+            </Directory>
+            <Directory Id="ProgramMenuFolder">
+                <Directory Id="AlacrittyProgramMenu" Name="Alacritty"/>
+            </Directory>
+        </Directory>
+
+        <!-- Application binaries -->
+        <DirectoryRef Id="AlacrittyProgramFiles">
+            <Component Id="AlacrittyExe" Guid="*">
+                <File Id="AlacrittyExeFile" Source="..\target\release\alacritty.exe" Name="alacritty.exe" KeyPath="yes"/>
+            </Component>
+            <Component Id="WinPtyAgentExe" Guid="*">
+                <File Id="WinPtyAgentExeFile" Source="..\target\release\winpty-agent.exe" Name="winpty-agent.exe" KeyPath="yes"/>
+            </Component>
+        </DirectoryRef>
+
+        <DirectoryRef Id="AlacrittyAppData">
+            <!-- Copy default config -->
+            <Component Id="AlacrittyConfig" Guid="*" Permanent="yes">
+                <File Id="AlacrittyConfigFile" Source="..\alacritty.yml" Name="alacritty.yml" KeyPath="yes"/>
+            </Component>
+        </DirectoryRef>
+
+        <DirectoryRef Id="AlacrittyProgramMenu">
+            <!-- Application shortcut -->
+            <Component Id="AlacrittyShortcut" Guid="aa36e61a-23cd-4383-b744-2f78e912f0dc">
+                <Shortcut Id="AlacrittyShortcutFile" Name="Alacritty" Description="A cross-platform, GPU-accelerated terminal emulator" Target="[AlacrittyProgramFiles]alacritty.exe"/>
+                <RemoveFolder Id="AlacrittyProgramMenu" On="uninstall"/>
+                <RegistryValue Root="HKCU" Key="Software\Microsoft\Alacritty" Name="installed" Type="integer" Value="1" KeyPath="yes"/>
+            </Component>
+        </DirectoryRef>
+
+        <DirectoryRef Id="AlacrittyProgramFiles">
+            <!-- Add to PATH -->
+            <Component Id="ModifyPathEnv" Guid="edf0b679-9eb6-46f7-a5d1-5160f30acb34" KeyPath="yes">
+                <Environment Id="PathEnv" Value="[AlacrittyProgramFiles]" Name="PATH" Permanent="no" Part="first" Action="set" System="yes"/>
+            </Component>
+        </DirectoryRef>
+
+        <DirectoryRef Id="TARGETDIR">
+            <!-- Add context menu -->
+            <Component Id="ContextMenu" Guid="449f9121-f7b9-41fe-82da-52349ea8ff91">
+                <RegistryKey Root="HKCU" Key="Software\Classes\Directory\Background\shell\Alacritty Here\command">
+                    <RegistryValue Type="string" Value="[AlacrittyProgramFiles]alacritty.exe --working-directory &quot;%V&quot;" KeyPath="yes"/>
+                </RegistryKey>
+                <RegistryKey Root="HKCU" Key="Software\Classes\Directory\Background\shell\Alacritty Here">
```

Comment:
I've tested this with powershell and it uses `Open PowerShell here`, we should follow this convention and change this to `Open Alacritty here`.

Reply:
Okay, sounds good.